winpr/libwinpr/input/keycode.c

```diff
@@ -450,6 +477,7 @@ DWORD KEYCODE_TO_VKCODE_EVDEV[256] =
 	0, /* <PAST> 143 */
 	0, /* <FIND> 144 */
 	0, /* <CUT> 145 */
+#endif
```

Comment:
These keys: "Stop", "Again", ..., "Paste", "Find", "Cut" are Sun keyboard native (those are placed on the left side below the "Help" key) and I suppose there are no such keys on "PC-104" kbd. It is a "heart issue" to find its re-mapping to the Virtual-Keys in freerdp to be able to map it back in case of the RDP client and also the server running both on Solaris. Currently my knowledge about these mappings in freerdp are very poor so please let me this small place-holder/reminder. :-)